### ISSUE TITLE
Problem for joint learning

### ISSUE THREAD

**Issue by yqdleiyi** [ppdebreuck/modnet#249]
I tried to implement joint learning with custom loss function, and I tried to add multiple loss function for multiple labels. I thought it should work well on MODNet since there is a good example on tf.keras.loss and it's the same in MODNet library. However, it could only work in simple MODNet model, not the model based on FitGenetic. What's wrong with it? The error for FitGenetic case is "no such type as list"
![image](https://github.com/user-attachments/assets/da458920-58dc-4d0f-b355-748f77ea7008)
![image](https://github.com/user-attachments/assets/e7e71e30-41a8-4f43-a350-46fad9d91cd8)
